### ISSUE TITLE
CrossMedia.Current.PickPhotoAsync in Android gives out of memory - picking a photo

### ISSUE THREAD

**Issue by micuentadecasa** [jamesmontemagno/MediaPlugin#101]
Please fill out either the bug or feature request section and remove whatever section you are not using.

## Bug 

Version Number of Plugin:
Device Tested On: Android
Simulator Tested On: --

### Expected Behavior
to fill an ImageSource with the picked image
### Actual Behavior
out of memory
 

I'm trying to use CrossMedia plugin in Android (picking from gallery) but gives me an out of memory when the image selected is big. The same code works ok on IOS.

   Stream stream = null;
            await CrossMedia.Current.PickPhotoAsync().ContinueWith(t =>
            {
                if (t.IsCompleted)
                {
                    MediaFile filex = t.Result;
                    stream = t.Result.GetStream();
                    pathOfImage = filex.Path;
                }
            }, TaskScheduler.FromCurrentSynchronizationContext()); 
            photi.Source = ImageSource.FromStream(() => stream);

I tried also to once known the path to resize the image before asigning it to the Image but when I try to load the file via System.IO it gives me an out of memory also.

If I choose a small image it works ok.

**Comment by jamesmontemagno** [comment]
What version of the plugin are you using? What device or emulator?

**Comment by micuentadecasa** [comment]
Hi @jamesmontemagno , I'm using 2.3.0 and Xamarin.Forms 2.0, the problem occurs in an Android device, but I have found a solution, I have created a custom renderer for the Image control, based on this article http://danielhindrikes.se/xamarin/xamarin-forms-and-large-images-in-android-apps/ . now with this extended control it works, it was  a problem with the Image control and not with this plugin. Sorry.

Note: I think that this information can be added as note for the people that uses your plugin on Android.
